REPORT.txt

```diff
@@ -1,18 +1,28 @@
 Why doesn't the starter code compile? How did you fix it?
 
-There is an issue with include guards, because Products are being defined twice. This is fixed by adding "#pragma once" to the top of product.h
+    There is an issue with include guards, because Products are being defined twice. This is fixed by adding "#pragma once" to the top of product.h
 
 What are the average and worst case time complexities for each map operation?
 
-O(1); O(length) in the worst case
+    O(1); O(length) in the worst case
 
-How many collisions are you getting with your chained implementaion? 
+How many collisions are you getting with your chained implementaion?
+
+    My first implementation has 2 collisions in name_to_id and 0 collisions in inventory.
 
 How long was the longest list of collisions?
 
+    The longest length was 2 (the shortest was 0).
+
 Describe exactly what you measured and how; consider making a new test case with more insertions.
 
```

Comment:
Please make a new test case which inserts a large number of elements, this will help you see how well your hash function works / how many collisions there are.